README.md

```diff
@@ -116,6 +116,21 @@ Install all the APKs, then start the Kitkit Launcher to explorer.
 - Please make sure there is enough storage on the device. 
 - Most of the APKs will not show up in the android (default) launcher. 
 
+
+# Trademarks
+Kitkit School, name and logo, and Enuma are trademarks of Enuma, Inc. The Apache License 2.0 under which Kitkit School software is released does not grant any license or right to use the trademarks. Please see Guidelines for Trademarks to learn more about the terms. (hyperlink to our website “Legal – Kitkit School” page)
+
+Kitkit and Enuma are trademarks of Enuma, Inc., registered in the U.S. and other countries. 
+
+
 # Questions?
 
 Do you have questions about the code, content or data? Please reach out to the [Global Learning XPRIZE Community](http://community.xprize.org/learning).
+
+
+<br>
```

Comment:
@todoschool HTML formatting does not work, as this is a Markdown document. (See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)